clients/src/main/java/org/apache/kafka/common/protocol/Writable.java

```diff
@@ -33,6 +35,23 @@
     void writeVarint(int i);
     void writeVarlong(long i);
 
+    default void writeApiMessage(
```

Comment:
It is used by only ```SendBuilder```. How about moving it to ```SendBuilder```?

Reply:
Fair enough.